commons/src/test/scala/dagr/commons/util/Tags.scala

```diff
@@ -21,14 +21,10 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.cmdline.parsing
 
-import dagr.core.util.UnitSpec
+package dagr.commons.util
 
-/**
-  * Integration tests for the command line parsing
-  */
-class CommandLineTest extends UnitSpec {
+import org.scalatest.Tag
 
-  // TOOD
-}
+/** Tags for our tests */
+object LongRunningTest extends Tag("LongRunningTest")
```

Comment:
Could we refactor a little here to:

1. Call this TestTags instead of just Tags
2. Place all (one) of our tag objects inside an `object TestTags` so that they don't clutter the namespace of the package as we add more?